packages/terra-clinical-item-view/src/ItemView.scss

```diff
@@ -11,20 +11,44 @@
     word-wrap: break-word;
   }
 
-  // Accessory is of fixed size and does not shrink or grow.
   .accessory {
+    display: flex !important;
     flex: 0 0 auto;
     margin: 5px;
-    max-height: 60px;
-    max-width: 60px;
     min-width: 0;
     overflow: hidden;
   }
 
   .accessory > * {
     display: flex !important;
-    max-height: 60px;
-    max-width: 60px;
+  }
+
+  // Start Accessory is of fixed size and does not shrink or grow.
+  .start-accessory,
+  .reserve-start-accessory-space {
+    justify-content: flex-start;
+    width: 2.8571rem;
+  }
+
+  .start-accessory > * {
+    max-height: 2.8571rem;
+    max-width: 2.8571rem;
+  }
+
+  .end-accessory,
+  .reserve-end-accessory-space {
+    justify-content: flex-end;
+    max-height: 7.14285rem;
```

Comment:
It seems weird to have different max heights between start and end. Maybe bump this one back to 2.8571 rem?

Reply:
Do we have a max height for the item view itself?

Reply:
There is no maximum height for the item view. I can bump it back to 2.8571 rems. I wasn't sure what might be the best here so gave it 100px

Reply:
Could we pin the hight of the start and end accessories to the hight of the item view based off the text height?

Reply:
When you say text height, are you thinking the height of the displays provided?

Reply:
yes + comment

Reply:
So, I did some research. Ultimately, flexbox doesn't allow you t create a row of flexbox items with a max height defined by one child. It looks like there could be possible work arounds using position absolute & relative, but this isn't a flexbox solution. One suggestion I read was to switch from a flexbox layout and use a tabular layout instead. Not sure how complex that could get with this implementation having flexboxs within flexboxs.

Reply:
@JakeLaCombe Do you have any insights here? I know you're pretty familiar with flexbox.